### PR TITLE
Use ColorMapper for event annotations

### LINKED ISSUE #10340: Enabling event annotations changes chart colors
When the event annotations are enabled for a chart, the color selection changes unnecessarily.

event-annotation-color-change

### PR DESCRIPTION
**Pull request by kmerz**
## Motivation
Before the change the event annotation color was not stable from the
beginning since it was not taking its color from the color mapper.

## Description
This PR will use the color mapper to add the color of the event
annotation.

Fixes #10340

## How Has This Been Tested?
- Create a histogram
- activate event annotation
- change the color of the event annotation

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: graylog2-web-interface/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "lint:styles": "stylelint './src/**/*.{js,jsx,ts,tsx}' --syntax css-in-js",
     "lint:styles:path": "stylelint --syntax css-in-js",
     "lint:changes": "./dev/lintChanges.sh",
-    "test": "jest --maxWorkers=50%",
+    "test": "jest --maxWorkers=3",
```
Comment (dennisoelkers) [inline graylog2-web-interface/package.json:24]: This line slipped in :/

**Review by dennisoelkers — CHANGES_REQUESTED** [review]
Looks good, works fine - just a single line that slipped in accidentally.

**Review by linuspahl — CHANGES_REQUESTED** [review]
It looks like the problem described in https://github.com/Graylog2/graylog2-server/issues/10340 still exists.

Untitled

**Comment by linuspahl** [comment]
@kmerz While the chart colors no longer change when:
- e.g. metric has no custom color and user enables event annotations
They are still changing when:
- metric has a custom color and user enables event annotations

event-annotations

**Review by linuspahl — APPROVED** [review]
Looks good to me! Could not find any errors. I thought it may results in a problem that we consider the old widget config when we submit the widget config form, but I could not find one.

**Review by dennisoelkers — APPROVED** [review]

**Pull request merged**